src/current/v24.1/user-defined-functions.md

```diff
@@ -14,7 +14,9 @@ A user-defined function (UDF) is a named function defined at the database level
 
 The basic components of a user-defined function are a name, list of arguments, return type, volatility, language, and function body.
 
-- An argument has a _mode_ and a _type_. CockroachDB supports the `IN` argument mode. The type can be a built-in type, [user-defined `ENUM`]({% link {{ page.version.version }}/enum.md %}), or implicit record type. CockroachDB **does not** support default values for arguments.
+- An argument has a _mode_ and a _type_.
+  - CockroachDB supports the `IN` (default), `OUT`, and `INOUT` argument modes. For an example, see [Create a function that uses `OUT` and `INOUT` parameters]({% link {{ page.version.version }}/create-function.md %}#create-a-function-that-uses-out-and-inout-parameters).
+  - The type can be a built-in type, [user-defined `ENUM`]({% link {{ page.version.version }}/enum.md %}), or implicit record type. A type can have a [`DEFAULT` value]({% link {{ page.version.version }}/default-value.md %}).
```

Comment:
Should we also add that user-defined composite types are allowed in addition to enums? Same for below.